app/src/main/java/com/internship/move/feature/onboarding/model/ViewPagerItem.kt

```diff
@@ -0,0 +1,9 @@
+package com.internship.move.feature.onboarding.model
```

Comment:
It seems that photo, heading and content fields are resources ids. Very good idea!

As a safety measure we annotate them with @DrawableRes, @StringRes, etc depending on what type of resource they are
This prevents anyone from sending a regular integer instead of a resource id, to the constructor